### DISCUSSION TITLE
Shelly Plus Add-on

### DISCUSSION THREAD

**Discussion by spiccinocchi** [RavenSystem/esp-homekit-devices#2285] [Q&A] [answered]
Is the Shelly Plus Add-On supported?

**Comment by RavenSystem** [comment]
Not yet. Soon. Support is on the way.

**Reply by spiccinocchi** [reply]
ok very good news!

**Reply by RavenSystem** [reply, marked as answer]
https://github.com/RavenSystem/esp-homekit-devices/releases/tag/HAA_12.13.3

**Reply by spiccinocchi** [reply]
It is possibile to manage a sensor state by attaching a switch to a Shelly Add-on?
if yes, witch is the right gpio and where the switch should be connected to the Add-on?

**Reply by RavenSystem** [reply]
DIGITAL IN = GPIO19

**Comment by wroblewski6** [comment]
Does it support add on on shelly plus or not ??

**Reply by wroblewski6** [reply]
I tried to use my shelly plus 1 pm with add on and ds18b20 temp sensor with this script 
{
  "t": 21,
  "n": "Ogrzewanie",
  "g": 21,
  "0": { "r": [[21]] },
  "1": { "r": [[21, 1]] },
  "dt": 1,
  "i2": 0,
  "f1": 5,
  "f2": 42,
  "f3": 0.5,
  "i": 0.2
}


And i see hester thermostat but all the time temp is -99 and can not fix it . Any idea what mistake i made ??

**Reply by RavenSystem** [reply]
https://github.com/RavenSystem/esp-homekit-devices/wiki/temperature-and-humidity-sensors#sensor-gpio

Replace `"g":21` with `"g":[1,0]`

And `"n": "Ogrzewanie"` does not make sense. Must be `"n".3`.

`"i2"` is useless. And f1, f2 and f3 are wrong. Same with "i": 0.2.

**Reply by wroblewski6** [reply]
Ok thx , will corect and check again . Chathpt made this mistakes
